.changeset/grumpy-toys-reply.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/cli": patch
+---
+
+- Removed descriptions adding for x-tagGroups for the `join` command. Descriptions in x-tagGroups are not supported and cause errors on linting.
+- `info.title` is now used as a name in x-tagGroups instead of a file name for the `join` command to make possible the joining of files with the same names.
```

Comment:
Will existing users see changed behaviour? Is it configurable?

Reply:
What if the `info.title`'s are the same?

Reply:
Use info versions as well? Is it a valid case to join different versions of the same API descriptions? If even that fails, I'd expect a warning/error. Alternatively, we can add indexes (like Title_1, Title_2 and so on), but this behaviour might be confusing.

Reply:
If the titles are the same, they'll only get one `x-tagGroups` entry. Let's link to the [info-override decorator](https://redocly.com/docs/cli/decorators/info-override/) in the join command docs so that the user can adjust one or all of the titles to suit their needs before joining.

Reply:
This docs change will be resolved in a separate PR here https://github.com/Redocly/redocly-cli/pull/1422